terraspace/.tool-versions

```diff
@@ -0,0 +1,3 @@
+nodejs 16.13.0
+golang 1.17.2
+ruby 3.1.1
```

Comment:
This file is missing a few other components that are used by the workflow files (`tfsec`, `tflint`, etc.). Please check my example branch for reference.

Reply:
My bad, I copied the wrong file instead of copying yours. Good eye!